torch/_ops.py

```diff
@@ -377,6 +377,12 @@ def check_overloaded(arg):
                     == torch._C._disabled_torch_dispatch_impl
                 ):
                     continue
+                if subclass_type is torch._subclasses.fake_tensor.FakeTensor:
+                    subclass_type = torch._subclasses.fake_tensor.FakeTensorMode  # type: ignore[assignment]
+                    handler = self.python_key_table[subclass_type]
+                    result = handler(arg.fake_mode, *args, **kwargs)  # type: ignore[attr-defined]
+                    return result
```

Comment:
@zou3519 Does this change looks ok to you?

Reply:
Here's the use case: during AOTI lowering, ppl wish to do numerical validation by running the lowered model eagerly. 

The model sometime has FakeTensor in it, for TBE weight. In such case, it will need the fixes here to make the eager run pass. It would produce a fake output. 

IIUC, you are suggesting we do a conditional setup for FakeMode. 

```
if model_has_fake_weights:
    with fake_mode_from_fake_weight: 
        model(args)
else:
    model(args)
```

Can we do this kind of check in our FakeTensor infra code? instead of asking user to set up the FakeMode?

Reply:
I'm convinced, thanks for the example. Can you leave a comment here saying something along the lines of "If people are using FakeTensor without a FakeTensorMode, pull the FakeTensorMode impl"?

Reply:
sure.